sdk/appcenter-data/src/test/java/com/microsoft/appcenter/data/DataTest.java

```diff
@@ -276,23 +278,23 @@ public void shouldNotOperateOnLocalStorageWhenPartitionNotExist() {
         /* Local storage create document should complete with not find partition error. */
         assertNotNull(doc);
         assertNotNull(doc.getError());
-        assertTrue(doc.getError().getMessage().contains(failedMessage));
+        assertTrue(Objects.requireNonNull(doc.getError().getMessage()).contains(failedMessage));
```

Comment:
```
    public static <T> T requireNonNull(T obj) {
        if (obj == null)
            throw new NullPointerException();
        return obj;
    }
```

That's what `Objects.requireNonNull` does so it does not make the test easy to read on failure for null (failed with exception instead of assert).

Prefer assertNotNull on previous line.